### ISSUE TITLE
Book 05: Sentence 099

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#13]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/0699e9cf-54e5-4713-8ea2-949fb9e15454)

```
# sent_id = Book_05_-_The_Omens.txt:99
# text = In chiqujmoli, cacentetl tototl, iuhqujn quauhchochopitli: auh veca quauhtla in jchan.
# text[norm] = In chiquimoli, ca centetl tototl, iuhquin cuauhchochopitli: auh huehca cuauhtla in ichan.
# text[orig] = I·nchiqujmoli·,·cacentetl·to·totl·,·iuh·qujn·quauh·chochopitli·:·auh·veca¶quauhtla·injchan·.
# text[eng] = The linnet, it is a bird, like the wood-pecker: but far in the forest is its home.
# text[osp] = Tambien tenjan por mal aguero, a las bozes del pito: [...]
# references = A&D/180:4
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 31.25%
1	In	in	DET|SCONJ	_	_	2	_	_	Orig=I·nchiqujmoli|Folio=13v,13v|Paragraph=6,6|Line=1,1|Norm=In
2	chiqujmoli	chiquimoli	NOUN	_	State=Abs	6	_	_	Orig=I·nchiqujmoli|Folio=13v,13v|Paragraph=6,6|Line=1,1|Norm=chiquimoli|Analysed=Yes
3	,	,	PUNCT	_		6	punct	_	Folio=13v|Paragraph=6|Line=1|Norm=,|Analysed=Yes
4-5	cacentetl	_	_	_	_	_	_	_	_
4	ca	cah|ca	VERB	_	_	6	_	_	Orig=I·nchiqujmoli|Folio=13v,13v|Paragraph=6,6|Line=1,1|Norm=ca
5	centetl	ce	NUM	_	NumType=Clf	6	_	_	Orig=I·nchiqujmoli|Folio=13v,13v|Paragraph=6,6|Line=1,1|Norm=centetl|Gloss=una.pieza|Split=cen·tetl
6	tototl	tototl	NOUN	_	State=Abs	0	root	_	Orig=to·totl|Folio=13v,13v|Paragraph=6,6|Line=1,1|Norm=tototl|Gloss=pájaro|Analysed=Yes
7	,	,	PUNCT	_		9	punct	_	Folio=13v|Paragraph=6|Line=1|Norm=,|Analysed=Yes
8	iuhqujn	iuhquin	ADV	_		9	_	_	Orig=iuh·qujn|Folio=13v,13v|Paragraph=6,6|Line=2,2|Norm=iuhquin|AmbigNorm=True|Override=False|Gloss=así|Analysed=Yes
9	quauhchochopitli	cuauhchochopitli	NOUN	_	State=Abs	6	_	_	Orig=quauh·chochopitli|Folio=13v,13v|Paragraph=6,6|Line=2,2|Norm=cuauhchochopitli|Analysed=Yes
10	:	:	PUNCT	_		13	punct	_	Folio=13v|Paragraph=6|Line=2|Norm=:|Analysed=Yes
11	auh	auh	CCONJ	_	_	13	_	_	Folio=13v|Paragraph=6|Line=2|Norm=auh|Gloss=y
12	veca	huehca	ADV	_		13	_	_	Folio=13v|Paragraph=6|Line=2|Norm=huehca|Gloss=lejos|Analysed=Yes
13	quauhtla	cuauhtla	NOUN	_	_	6	_	_	Folio=13v|Paragraph=6|Line=3|Norm=cuauhtla|Gloss=bosque
14	in	in	DET|SCONJ	_	_	15	_	_	Orig=injchan|Folio=13v|Paragraph=6|Line=3|Norm=in
15	jchan	chantli	NOUN	_	Number[psor]=Sing|Person[psor]=3|State=Poss	13	_	_	Orig=injchan|Folio=13v|Paragraph=6|Line=3|Norm=ichan|Gloss=su.casa|Analysed=Yes
16	.	.	PUNCT	_		6	punct	_	Folio=13v|Paragraph=6|Line=3|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
Questions:

- Treatment of *in* when followed by a noun, or "do we accept a determiner reading of "in"
- What is the subject? *centetl tototl*, if so, what is *in chiquimoli*? `dislocated` ? Or is *In chiquimoli* the subject and *centetl tototl* the predicate?
- What should we do with *ca* ?  Is it a copula, or a verb, if it is a copula do we treat it with `cop`, or should we do VERB + `xcomp` for the predicate and then convert it for publishing in official UD? 
- Is the analysis of the numeral centetl good, e.g. it's a numeral + classifier, are there other classifiers, how many, is this a good way to analyse them?
- What shall we do with *iuhquin cuauhchochopitli*, if *iuhquin* is modifying *cuauhchochopitli* a `NOUN`/`case` then it should be `ADP`, but we probably don't like that (see `answers-20111108082304AAEbrNs_ans-0010` in UD_English-EWT train)
- For the conjoined clause, *in ichan* is the subject and *huehca cuauhtla* is the predicate?

**Comment by Lguyogiro** [comment]
> Treatment of in when followed by a noun, or "do we accept a determiner reading of "in"

Yeah I think we must accept the determiner reading pre-nominally unless it is obviously a clause (doesn't seem like the case here).

> What is the subject? centetl tototl, if so, what is in chiquimoli? dislocated ? Or is In chiquimoli the subject and centetl tototl the predicate?

This is an equational type construction, where _chiquimoli_ is preceded by `in`, so I think _centetl tototl_ is good as the root, and _chiquimoli_ is the subj.

> What should we do with ca ? Is it a copula, or a verb, if it is a copula do we treat it with cop, or should we do VERB + xcomp for the predicate and then convert it for publishing in official UD?

Generally I like treatment of _ca_ as `AUX`, so maybe in this case `cop`, and then `aux` when it precedes a verb? Not sure.

**Comment by ftyers** [comment]
> Generally I like treatment of ca as AUX, so maybe in this case cop, and then aux when it precedes a verb? Not sure. (I think elsewhere we said it has a discourse relation, which, on thinking a bit more about, I'm not sure I'm convinced of).

Is this *cah* or *ca*? I was assuming it was *ca*. If this is *cah* then it should probably be `AUX`/`cop`.

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/2386bbf0-8cd9-4e9e-94c3-9c609838ba9a)

From the docs:

`discourse`:
> This is used for interjections and other discourse particles and elements (which are not clearly linked to the structure of the sentence, except in an expressive way). We generally follow the guidelines of what the Penn Treebanks count as an INTJ. They define this to include: interjections (oh, uh-huh, Welcome), fillers (um, ah), and discourse markers (well, like, actually, but not you know).

`aux`:
> An aux (auxiliary) of a clause is a function word associated with a verbal predicate that expresses categories such as tense, mood, aspect, voice or evidentiality. It is often a verb (which may have non-auxiliary uses as well).

I'd say this is closer to `PART`/`discourse` than to `AUX`/`aux`.

What does Lockhart say?

**Comment by Lguyogiro** [comment]
Yeah, I guess if they say _actually_ is `discourse`, we should do the same with _ca_

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/0d3f80ba-164e-44a7-8e2d-3ed1b023c305)

Remaining questions:
- What is the POS of *iuhquin* and what is its relation to *cuauhchochopitli*? 
- What is the structure of *iuhquin cuauhchochopitli* with respect to *tototl*?

**Comment by Lguyogiro** [comment]
![image](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/55f0ae19-c557-4f23-8ad1-e002f8c667a5)


I'm not sure about the upos of _iuhquin_...I think maybe `ADJ`. In the analysis here, I'm assuming of gloss of e.g. "a bird that is similar to a woodpecker"

**Comment by ftyers** [comment]
Some useful information from Launey:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/0d2ec6ad-cbb0-4056-9c38-b33b01e52665)

And Andrews:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/468cecc3-af41-46c0-b9df-6d319503ce25)

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/43184ab9-3c8a-47fb-a8e6-cd24134d4c57)

Questions/comments:
- Do we potentially want to split this into *iuhqui in*? 
- According to the above, *cuauhchochopitli* is a predicate (e.g. it takes the person agreement)
- *cuauhchochopitli* is subordinated to *iuhqui* (because of the *in*)
- *iuhqui* is some kind of nominal 
- If we go with "being thus a woodpecker", then maybe `xcomp(iuhqui, cuauhchochopitli)`, `acl(tototl, iuhqui)`
  - But then we have the question of the POS for *iuhqui*, `ADJ` or `VERB`

**Comment by Lguyogiro** [comment]
> If we go with "being thus a woodpecker", then maybe xcomp(iuhqui, cuauhchochopitli), acl(tototl, iuhqui)
> 
>     But then we have the question of the POS for iuhqui, ADJ or VERB
> 

I think this makes sense. re POS, I vote for `ADJ` (like one such as thus) or `NOUN` (one such as thus).

Its also interesting to note that this is the origin of _ihquin/ihcon/ohcon_, which in azz and nhi are `ADV`

**Comment by ftyers** [comment]
> I think this makes sense. re POS, I vote for `ADJ` (like one such as thus) or `NOUN` (one such as thus).

In that case, do we want to split the *in* off ?

(a) 
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/8d80a10e-2c70-4763-8733-ea3d1d61f374)

or leave it:

(b)
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/4229ac1e-710a-4f5e-822e-425838a7caff)

There may be other stuff with the *iuh-* stem that we should be careful to treat in a similar way.

**Comment by Lguyogiro** [comment]
I kind of think we should split off _in_. In cases of iuhqui without the word-final _n_, the next word is always _in_.

eg 
`iuhquj in qujmolpilitica`

also, shouldn't the surface form be _n_ instead of _in_? i.e. iuhquijn -> iuhquj n

**Comment by ftyers** [comment]
> also, shouldn't the surface form be _n_ instead of _in_? i.e. iuhqujn -> iuhquj n

It could go either way, but sure we can do it like that. 

So:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/a26feef7-5564-4b55-ab9a-4a756dc1d578)

**Comment by Lguyogiro** [comment]
:+1:

**Comment by ftyers** [comment]
Closed in 4a37124.
